splash/pages/pages_routes.py

```diff
@@ -95,21 +105,30 @@ def get_pages_by_type(
     return results
 
 
-@pages_router.put("/{uid}", tags=["pages"], response_model=CreatePageResponse)
+@pages_router.put(
+    "/{uid}",
+    tags=["pages"],
+    response_model=CreatePageResponse,
+)
 def replace_page(
-    uid: str, page: UpdatePage, current_user: User = Security(get_current_user)
+    uid: str,
+    page: UpdatePage,
+    response: Response,
```

Comment:
why is response an un-used import parameter?

Reply:
That is an accident on my part.